src/components/common/index.js

```diff
@@ -1,6 +1,6 @@
 import { css } from "styled-components";
 
-export const shadow = `4px 4px 10px 0 rgba(0, 0, 0, 0.35)`;
+export const shadow = `4px 4px 10px 0 rgba(0,0,0,0.35)`;
 export const insetShadow = `inset 3px 3px 10px rgba(0, 0, 0, 0.2)`;
```

Comment:
I think you should revert these formatting changes

Reply:
Sure thing. They were made in response to the following test failure I kept getting. 

    Component: AppBar › renders default component styles

    "Value mismatch for property 'box-shadow'"

    Expected
      "box-shadow: 4px 4px 10px 0 rgba(0, 0, 0, 0.35),inset 1px 1px 0px 1px #dfe0e3,inset -1px -1px 0 1px #888c8f"
    Received:
      "box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.35),inset 1px 1px 0px 1px #dfe0e3,inset -1px -1px 0 1px #888c8f"

But if they aren't that valuable I'll go ahead revert them.

Reply:
We can keep that. We will have to change the insetShadow later on too if we wanna test for styles (text inputs and Cutout uses this variable)